lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -8,7 +8,7 @@ module ClassMethods
         # Comfy already define #tags and #tags= methods in some models, so let's call :keywords to the Tag associated
         # instances of a model to avoid naming collisions!!!
         # Creates the appropriate AR associations, scopes and instance methods.
-        def is_taggable
+        def is_taggable # rubocop:disable Style/PredicateName
```

Comment:
@benlovell `is_taggable` here isn't a boolean method, which took me a little while to figure out. It actually does a bunch of wiring up (I think), so it's more like... in the imperative sense? So it's more like saying "this object is taggable" rather than asking "is this object taggable?" It's weird, I know.

@tomas-stefano My experience is that warnings which stick around longer than a certain time have the effect of undermining the styleguide. It's harder to notice regressions and there's less of a psychological imperative to 'not break it' (because it already doesn't pass). Having a fully green codebase would also let us run automated build-failing tasks like the one in your pre-commit hook, thereby outright preventing further regressions.

It's your call though — feel free to close the PR if it's not the approach you want to take.